core/deployment/src/main/java/io/quarkus/deployment/pkg/steps/NativeImageBuildContainerRunner.java

```diff
@@ -30,14 +31,18 @@ public abstract class NativeImageBuildContainerRunner extends NativeImageBuildRu
     public NativeImageBuildContainerRunner(NativeConfig nativeConfig, Path outputDir) {
         this.nativeConfig = nativeConfig;
         containerRuntime = nativeConfig.containerRuntime.orElseGet(ContainerRuntimeUtil::detectContainerRuntime);
-        log.infof("Using %s to run the native image builder", containerRuntime.getExecutableName());
```

Comment:
I moved this to the build step in order to preserve the same relative order of logs.